Content.Shared/NPC/Components/FactionExceptionComponent.cs

```diff
@@ -7,7 +7,7 @@ namespace Content.Shared.NPC.Components;
 /// Prevents an NPC from attacking ignored entities from enemy factions.
 /// Can be added to if pettable, see PettableFriendComponent.
 /// </summary>
-[RegisterComponent, NetworkedComponent, Access(typeof(NpcFactionSystem), typeof(SharedNPCImprintingOnSpawnBehaviourSystem))] // TO DO (Metalgearsloth): If we start adding a billion access overrides they should be going through a system as then there's no reason to have access, but I'll fix this when I rework npcs.
```

Comment:
theres an api already @TheShuEd why didnt you use it noob

Reply:
???

Reply:
use api instead of bypassing it with access

Reply:
Idk how

Reply:
public methods on NpcFactionSystem